tests/rest/client/v2_alpha/test_account.py

```diff
@@ -458,6 +458,29 @@ def deactivate(self, user_id, tok):
         self.assertEqual(channel.code, 200)
 
 
+class WhoamiTestCase(unittest.HomeserverTestCase):
+
+    servlets = [
+        synapse.rest.admin.register_servlets_for_client_rest_resource,
+        login.register_servlets,
+        account.register_servlets,
+        register.register_servlets,
+    ]
+
+    def test_GET_whoami(self):
+        device_id = "wouldgohere"
+        user_id = self.register_user("kermit", "test")
+        tok = self.login("kermit", "test", device_id=device_id)
+
+        whoami = self.whoami(tok)
+        self.assertEqual(whoami, {"user_id": user_id, "device_id": device_id})
```

Comment:
I think this should be using `assertDictEqual` instead of `assertEqual`.

Reply:
'assertEqual' automatically uses that for dicts.

Reply:
Oh my bad, ignore me then, thanks for the tip!